### PR TITLE
Social metadata

### PR DESCRIPTION
**Pull request by jakearchibald**

### CODE REVIEW

File: confbox.config.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
 module.exports = {
+  /**
+   * Origin of the conference, for creating absolute URLs.
+   */
+  origin: 'https://social--cds2019.netlify.com',
```
Comment (surma) [inline confbox.config.js]: I mean, we shouldn’t merge it like this, should we?

File: src/_includes/layouts/main/index.njk [hunk 2]
```diff
@@ -6,12 +6,41 @@
...
+    {% else %}
+      <meta property="og:title" content="{{conf.conferenceName}}">
+      <meta name="twitter:title" content="{{conf.conferenceName}}">
+    {% endif %}
+
+    {% if description %}
+      <meta name="description" content="{{description}}">
+      <meta property="og:description" content="{{description}}">
+      <meta name="twitter:description" content="{{description}}">
+    {% endif %}
```
Comment (surma) [inline src/_includes/layouts/main/index.njk]: Should we provide a default description?

File: confbox.config.js (hunk 1 above)
Comment (jakearchibald) [reply confbox.config.js]: haha shit I knew I'd forget about this

File: src/_includes/layouts/main/index.njk (hunk 2 above)
Comment (jakearchibald) [reply src/_includes/layouts/main/index.njk]: We should try and have a description on every page, but I don't think a default description is better than what Facebook/Twitter will infer from the page.

Comment (argyleink) [reply src/_includes/layouts/main/index.njk]: sounds like a great thing to punt to our marketing copy team? let them know we want a default description, but also 1 for each page: contextual page descriptions for social share, love that y'all're on top of this 👍

File: src/index.njk [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ---
+description: Connect with Chrome engineers and other leading developers for an exploration of building beautiful and performant experiences.
```
Comment (argyleink) [inline src/index.njk]: this text should pair with whatever social image we choose from #154, which sounds like it should include location and other RSVP type details?

**Pull request merged**